.github/workflows/update-events.yml

```
@@ -0,0 +1,63 @@
+name: "Actualizar eventos meetup"
+
+on:
```

@Scot3004 puedes probar con este cambio para ejecutar el job antes de mezclarlo?
O lo probaste en algun otro repositorio propio?

```suggestion
on:
  pull_request:
    branches:
      - develop
```

---

No lo quise poner, la idea es que la actualización de eventos se haga bajo una de estas 2 premisas
* Una vez al mes para actualizar según lo que este en meetup, para que no se nos quede tan atras como ahora que se queda un año y nadie envia un pr actualizando eventos
* A demanda, por ejemplo si quiero promocionar un evento en mi grupo local y lo acabo de crear, con este dispatch puedo forzar que se actualicen los eventos

En cuanto al por que no considero conveniente que se corra cuando se hace un pull request, cada vez que se haga un pull request se creara otro con los eventos actualizados, a decir verdad para eso tenemos `deploy.yml`  

En si este job solo sirve para actualizar los eventos es algo así como una etl que extrae la data de meetup, la transforma en content.lr y luego lo carga en github

---

Aca está el ejemplo de como deberia funcionar

https://github.com/PyBAQ/website/actions/workflows/actualizar-eventos-meetup.yml

Ten en cuenta que tiene ciertas diferencias, pero en esencia hacen lo mismo